packages/integrations/markdoc/package.json

```diff
@@ -32,19 +32,23 @@
   },
   "dependencies": {
     "@markdoc/markdoc": "^0.2.2",
+    "astro": "workspace:*",
```

Comment:
Note: a direct dependency on Astro _will_ cause changesets to auto-bump the Markdoc package on core changes. This is because of a direct dependency on the `emitESMImage` utility function. We should discuss if this is acceptable for an experimental integration or not.

Reply:
Hmm, I know you gave two options when making this, but would it be perhaps better to pass `emitESMImage` somehow?

Reply:
Yeah this bothers me... I can pass down `emitESMImage` via the `getRenderModule()` context on `this`. Just feels a bit dirty

Reply:
If you make it be a peerDependency it won't bump automatically. All of the other integrations are peerDeps. Is there a reason for this one to have a direct?

Reply:
Moved back to a peer dep! It's only pulled in to access an experimental function (`emitESMImage`), so as long as this is tracked, we should be safe to use peer deps alone.